### PR TITLE
license added to about page.

### PR DESCRIPTION
**Pull request by githubashutoshsoni**
Added license page @gavv can you review please?

### CODE REVIEW

**Comment by gavv** [comment]
#51

**Comment by gavv** [comment]
Thanks for PR! I just created the task, the maintainer of roc-droid is @ortex, he does the review :)

File: app/src/main/java/org/rocstreaming/rocdroid/LibraryActivity.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_library)
+
+        val recyclerView = findViewById<RecyclerView>(R.id.licenseRecyclerView)
+        val adapter = LicenseAdapter(arrayList)
+        recyclerView.layoutManager = LinearLayoutManager(this)
+        recyclerView.adapter = adapter
+    }
+
+    val arrayList = listOf<License>(License("roc-droid", "org.roc-streaming.roctoolkit:roc-android:0.2.1"))
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/LibraryActivity.kt:21]: goal was to somehow (probably with gradle plugin e.g. https://github.com/jk1/Gradle-License-Report) collect dependencies and their licenses automatically, not hardcode it

**Comment by JamesOsborn-SE** [comment]
You should be able to add the license file to assets and do something like this: https://github.com/JamesOsborn-SE/eat-poop-you-cat-android/blob/6d75740d57687db2c1365bda17b3745f5f945e67/app/src/main/java/dev/develsinthedetails/eatpoopyoucat/utilities/ReadMetadata.kt

**Comment by gavv** [comment]
@JamesOsborn-SE What I meant in the issue is to add licenses of all dependencies, not just license of roc droid, because some of them may require attribution. And I guess it would be nice to do it automatically because dependencies can change. Though I guess this feature is not that important.

**Comment by gavv** [comment]
@githubashutoshsoni Hey, do you have any plans on this PR, or should we close it?

**Comment by githubashutoshsoni** [comment]
> @githubashutoshsoni Hey, do you have any plans on this PR, or should we close it?

No, I’m busy with my job right now. So don’t have the time to do it.

**Pull request closed without merging**